pandas/plotting/_core.py

```diff
@@ -889,6 +909,68 @@ def _make_plot(self):
             ax.errorbar(data[x].values, data[y].values,
                         linestyle='none', **err_kwds)
 
+    def _sci_notation(self, num):
+        """
+        Returns mantissa and exponent of the number passed in argument.
+        Example:
+        >>> _sci_notation(89278.8924)
+        (8.9, 5.0)
```

Comment:
This returns (8.9, 4.0) for me - minor typo?

Reply:
You're right.